tracker/view/show.py

```diff
@@ -439,13 +443,17 @@ def show_package(pkgname):
                            package=data)
 
 
-def render_html_advisory(advisory, package, group, raw_asa, generated):
+def render_html_advisory(advisory, package, group, raw_asa, generated, form=None, reviews=[]):
```

Comment:
initializing to a default list is a way to shoot yourself in the foot. I'd advice to change reviews to None and add a guard for it.